lux/_config/config.py

```diff
@@ -26,6 +26,7 @@ def __init__(self):
         self._sampling_cap = 30000
         self._sampling_flag = True
         self._heatmap_flag = True
+        self._set_vislib = "vegalite"
```

Comment:
Could we rename this to `plotting_backend` so that it uses the same terminology as [pandas's `pd.options.plotting.backend`](https://plotly.com/python/pandas-backend/)?
So users can change the visualization library used via `lux.config.plotting_backend='matplotlib'`.